notes/2023-09-26.md

```diff
@@ -40,4 +40,8 @@ References until the full notes are added:
 
 Use an issue to submit a question for today, or make a PR to post a question and the best answer you can find (I'll review and fix if needed before posting) for a community badge.  
 
+Q: My question for today's class is if you can use reset to go forward in time commit-wise. For example, if you reset too far back, can you go back to a more recent commit.
+
+A: The best answer I can find is that you can revert back to a more recent commit after resetting to an older one. I also read that there is a chance git does some cleanup after a few days and deltes the commits that come after the time of the reset, making going forward in time impossible. This might only be in edge cases though. 
+
 ```
```

Comment:
````suggestion
```
### Can use reset to go forward in time commit-wise. For example, if you reset too far back, can you go back to a more recent commit?

The best answer I can find is that you can revert back to a more recent commit after resetting to an older one. I also read that there is a chance git does some cleanup after a few days and deltes the commits that come after the time of the reset, making going forward in time impossible. This might only be in edge cases though. 


````

Reply:
syntax fixes